tests/integration/test_credentials.py

```diff
@@ -72,7 +72,10 @@ def test_access_secret_vs_profile_code(self, credentials_cls):
         )
 
         credentials_cls.assert_called_with(
-            access_key='code', secret_key='code-secret', token=mock.ANY
+            access_key='code',
+            secret_key='code-secret',
+            token=mock.ANY,
+            account_id=mock.ANY,
```

Comment:
We can avoid making this change by only setting account ID in `ConfigProvider.load` if the value isn't `None`. However, we haven't conditionally provided values to a credentials object before so I elected not to for the time being.